Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "xmake"
-version = "0.2.2"
+version = "0.2.4"
```

Comment:
You can leave the `0.2.3`, the PR will count as one patch anyway

Reply:
Okay, I will soon merge these commits into single one.